src/rgw/rgw_op.cc

```diff
@@ -7850,6 +7850,13 @@ int RGWDeleteBucketPolicy::verify_permission()
 
 void RGWDeleteBucketPolicy::execute()
 {
+  bufferlist data;
+  op_ret = forward_request_to_master(s, nullptr, store, data, nullptr, &s->info);
```

Comment:
all of the other callers (at least, on octopus) are making this call conditionally. for example, in `RGWPutBucketPolicy::execute()`:
```c++
  if (!store->svc()->zone->is_meta_master()) {
    op_ret = forward_request_to_master(s, NULL, store, data, nullptr);
    if (op_ret < 0) {
      ldpp_dout(this, 20) << "forward_request_to_master returned ret=" << op_ret << dendl;
      return;
    }
  }
```
can you please wrap both of these calls to `forward_request_to_master()` the same way?
```c++
  if (!store->svc()->zone->is_meta_master()) {
```